src/js/plugins/plugin.mobile_nav.js

```diff
@@ -148,10 +178,20 @@ BookReader.prototype.buildToolbarElement = (function (super_) {
  */
 BookReader.prototype.buildMobileDrawerElement = function() {
   let experimentalHtml = '';
+  //builds filters checkbox html
   if (this.enableExperimentalControls) {
     experimentalHtml = `
         <p class="DrawerSettingsTitle">Experimental (may not work)</p>
-        <button class="BRaction default high-contrast-button">Toggle high contrast</button>
+        <div class="BRcheckbox-group-filters">
+          <input type="checkbox" class="BRcheckbox-filters" id="filter1">
+          <label for="filter1" class="BRcheckbox-label-filters">Grayscale</label><br>
+          <input type="checkbox" class="BRcheckbox-filters" id="filter2">
+          <label for="filter2" class="BRcheckbox-label-filters">High brightness</label><br>
+          <input type="checkbox" class="BRcheckbox-filters" id="filter3">
+          <label for="filter3" class="BRcheckbox-label-filters">Inverted</label><br>
+          <input type="checkbox" class="BRcheckbox-filters" id="filter4">
+          <label for="filter4" class="BRcheckbox-label-filters">High contrast</label><br>
```

Comment:
It would be nice if we could DRY this up, since there's a bit of duplicated code. Maybe converting `FILTERLIST` to an array of objects, over which we can iterate here and build each checkbox.

Reply:
Oh, we also don't need the `for=` attrs if we do something like:

```html
<label class="BRcheckbox-label-filters"><input type="checkbox"> High contrast </label><br> 
```

This will make clicking the text check/uncheck the checkbox